src/main/java/Command.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * The Command class makes use of multiple classes to drive the program.
+ */
+public class Command {
+
+    private String command;
+
+    /**
+     * Creates a Command object.
+     *
+     * @param command A string representation of the user's command.
+     */
+    public Command(String command) {
+        this.command = command;
+    }
+
+    /**
+     * Responds accordingly to the user's command.
+     *
+     * @param tasks A TaskList object containing the list of tasks.
+     * @param ui An Ui object that helps to interact with the user.
+     * @param storage A Storage object that contains a file with the tasks in it.
+     */
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
```

Comment:
Since commands are limited to a fixed set of values, maybe it would be a good idea to use an enum?

I think it also offers a trivial runtime benefit as you only call CommandsEnum.valueOf(taskName) once ( and then catch IllegalArgumentException if invalid), and compare for enum equality.
( In C / C++ / C# enums are fundamentally implemented with integers, I believe it is the same for Java but I cant find a good source on it thus far )

(instead of comparing string equality till the correct string is reached)